skins/tailwindui/layouts/_head_auth.php

```diff
@@ -28,13 +28,13 @@
 ?>
 
 <?php foreach ($styles as $style) : ?>
-    <link href="<?= $style . '?v=' . $coreBuild; ?>" rel="stylesheet" importance="high">
-    <link href="<?= $style . '?v=' . $coreBuild; ?>" rel="preload" as="style" importance="high">
+    <link rel="preload" href="<?= $style . '?v=' . $coreBuild; ?>" as="style">
+    <link rel="stylesheet" href="<?= $style . '?v=' . $coreBuild; ?>">
 <?php endforeach; ?>
 
 <?php foreach ($scripts as $script) : ?>
-    <script data-cfasync="false" src="<?= $script . '?v=' . $coreBuild; ?>" importance="high"></script>
```

Comment:
data-cfasync is required to work well with CloudFlare AFAIK

Reply:
Discussed at https://github.com/wintercms/winter/pull/1014#discussion_r1430729143